apps/src/applab/dontMarshalApi.js

```diff
@@ -180,6 +180,27 @@ export function appendItem(
   }
 }
 
+function interpreterGetValue(object, key) {
+  return getValue(object, key, true);
+}
+
+export function getValue(object, key, calledWithinInterpreter) {
+  const objectContents = calledWithinInterpreter ? object.properties : object;
+  const keyData = key.data;
+  return objectContents[keyData].data;
+}
+
+function interpreterAddPair(object, key, value) {
```

Comment:
nit/suggestion: Do you want to do any validation about the params passed in? for example, `key` should be a string, right? https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/util/javascriptMode.js#L38 makes it easy to output a warning to the console if the user passes in parameters with the wrong types.
also fine to leave as future work!

Reply:
Added in validation that keys are strings. Didn't validation value since it can be pretty much any type.